friends/friendModel.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+const FriendSchema = new mongoose.Schema({
+	firstName: {
+		type: String,
+		required: true
+	},
+	
+	lastName: {
+		type: String,
+		required: true
+	},
+	
+	age: {
+		type: Number,
+		required: true,
+		min: [1],
+		max: [120]
```

Comment:
I think technically you don't _need_ the min/max here because you're validating age within your catch, but it's still (IMO) good practice to be as specific as possible about requirements.